perseus-api/perseus_api.py

```diff
@@ -73,9 +81,9 @@ def create_source_schema_by_scan_report(current_user):
     app.logger.info("REST request to upload scan report from file manager and create source schema")
     try:
         scan_report = scan_report_request.from_json(request.json)
-        scan_reports_service.load_scan_report_from_file_manager(scan_report, current_user)
-        saved_schema = source_schema_service.create_source_schema_by_scan_report(current_user, scan_report.file_name)
-        etl_mapping = etl_mapping_service.create_etl_mapping_from_request(current_user, scan_report)
+        load_scan_report_from_file_manager(scan_report, current_user)
+        saved_schema = create_source_schema_by_scan_report(current_user, scan_report.file_name)
```

Comment:
"create_source_schema_by_scan_report" same as the parent method name. Will be easily to call method via service name